bin/cron/deploy_to_test

```diff
@@ -31,6 +31,31 @@ def main
 
   DevelopersTopic.set_dtt TOPIC_DTT_IN_PROGRESS
 
+  unless GitHub.compare(base: 'staging', head: 'test').empty?
+    unless DevelopersTopic.dts?
+      ChatClient.message(
+        'infra-test',
+        'robo-DTT skipped (DTS was not yes, with DTS required)',
+        color: 'yellow'
+      )
+      return
+    end
+    ChatClient.message(
+      'infra-test',
+      'robo-DTT is merging Test > Staging',
+      color: 'green'
+    )
+    pr_number = GitHub.create_and_merge_pull_request(
+      base: 'staging',
+      head: 'head',
+      title: 'DTS (Test > Staging) [robo-dtt]'
+    )
+    if pr_number.nil?
+      raise Exception.new('GitHub.create_and_merge_pull_request failed.')
+    end
+    sleep 15
```

Comment:
It is totally unclear to me whether this sleep is required. It just seems useful so as to not have a race condition with the GitHub API. Thoughts?